content/docs/setup/kubernetes/additional-setup/mesh-expansion/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Mesh Expansion
```

Comment:
I think that now that this is more thorough, it shouldn't be buried under `additional-guides`. Should we move it to `docs/examples/mesh-expansion/_index.md`? Alternatively, maybe put it under install: `docs/setup/kubernetes/install/mesh-expansion/_index.md`, but it's really more of an example than installation instructions IMO.

@geeknoid, @rcaballeromx  WDYT?

Reply:
@frankbu I agree that it shouldn't be under `additional-guides`
I would suggest placing the content under `/docs/tasks/mesh-expansion/`

Reply:
Agreed, but that's `/docs/examples/mesh-expansion/`. There are no setup type tasks.

Reply:
Moved to docs/examples/mesh-expansion/